src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -376,41 +352,66 @@ export class RbdFormComponent implements OnInit {
     };
   }
 
+  protected getDependendChildFeatures(featureKey: string) {
```

Comment:
(Found by my IDE ;) )
```suggestion
  protected getDependentChildFeatures(featureKey: string) {
```